experimental/devices/epd/epd.go

```diff
@@ -0,0 +1,463 @@
+// Copyright 2018 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+package epd
+
+import (
+	"errors"
+	"fmt"
+	"image"
+	"image/color"
+	"image/draw"
+	"time"
+
+	"periph.io/x/periph/conn"
+	"periph.io/x/periph/conn/display"
+	"periph.io/x/periph/conn/gpio"
+	"periph.io/x/periph/conn/gpio/gpioreg"
+	"periph.io/x/periph/conn/physic"
+	"periph.io/x/periph/conn/spi"
+	"periph.io/x/periph/devices/ssd1306/image1bit"
+)
+
+// Raspberry pi hat pins
+const (
+	ResetPin       = "17"
+	DataCommandPin = "25"
+	ChipSelectPin  = "8"
+	BusyStatePin   = "24"
+)
+
+// EPD commands
+const (
+	DriverOutputControl            = 0x01
+	BoosterSoftStartControl        = 0x0C
+	GateScanStartPosition          = 0x0F
+	DeepSleepMode                  = 0x10
+	DataEntryModeSetting           = 0x11
+	SwReset                        = 0x12
+	TemperatureSensorControl       = 0x1A
+	MasterActivation               = 0x20
+	DisplayUpdateControl1          = 0x21
+	DisplayUpdateControl2          = 0x22
+	WriteRAM                       = 0x24
+	WriteVcomRegister              = 0x2C
+	WriteLutRegister               = 0x32
+	SetDummyLinePeriod             = 0x3A
+	SetGateTime                    = 0x3B
+	BorderWaveformControl          = 0x3C
+	SetRAMXAddressStartEndPosition = 0x44
+	SetRAMYAddressStartEndPosition = 0x45
+	SetRAMXAddressCounter          = 0x4E
+	SetRAMYAddressCounter          = 0x4F
+	TerminateFrameReadWrite        = 0xFF
+)
+
+// LUT contains the display specific waveform for the pixel programming of the display.
+type LUT []byte
+
+// LUTType full or partial display update
+type LUTType int
```

Comment:
Unless there's other types, a bool would be fine, like `PartialUpdate bool`

Reply:
I changed the name of this type alias but I don't know if it became better to understand if someone takes a look at the code. If you can give more ideas on that would be nice.